lxd/api_internal.go

```diff
@@ -123,11 +126,64 @@ var internalImageOptimizeCmd = APIEndpoint{
 	Post: APIEndpointAction{Handler: internalOptimizeImage},
 }
 
+var internalWarningCreateCmd = APIEndpoint{
+	Path: "testing/warnings",
+
+	Post: APIEndpointAction{Handler: internalCreateWarning},
+}
+
 type internalImageOptimizePost struct {
 	Image api.Image `json:"image" yaml:"image"`
 	Pool  string    `json:"pool" yaml:"pool"`
 }
 
+type internalWarningCreatePost struct {
+	Location       string `json:"location" yaml:"location"`
+	Project        string `json:"project" yaml:"project"`
+	EntityTypeCode int    `json:"entity_type_code" yaml:"entity_type_code"`
+	EntityID       int    `json:"entity_id" yaml:"entity_id"`
+	TypeCode       int    `json:"type_code" yaml:"type_code"`
+	Message        string `json:"message" yaml:"message"`
+}
+
+func internalCreateWarning(d *Daemon, r *http.Request) response.Response {
```

Comment:
@monstermunchkin as this is a testing route, is it used to artificially create a warning? Would be good to have a comment here to that effect for future clarity.